Assets/Scripts/Enemy/EnemyFactory.cs

```diff
@@ -50,7 +50,7 @@ public EnemyFactory()
     {
         // Define enemyInfo for each type.
         AddTurretToRoster();
-        _enemyInfo.Add(EnemyType.GRENADIER, new EnemyInfo(EnemyMovements.GrenadierMovement, Globals.DirectTargeting, EnemyMovements.GrenadierSetup, new ArtilleryAttackBehaviour(EntityType.ENEMY, 20f), 10f));
+        _enemyInfo.Add(EnemyType.GRENADIER, new EnemyInfo(EnemyMovements.GrenadierMovement, Globals.GrenadierTargeting, EnemyMovements.GrenadierSetup, new ArtilleryAttackBehaviour(EntityType.ENEMY, 20f), 1f));
```

Comment:
we probably want to revert the attack speed back or make it a bit slower

Reply:
Period is now 3s.